src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/DisputantUpdateRequestStatus.java

```diff
@@ -4,6 +4,7 @@ public enum DisputantUpdateRequestStatus implements ShortNamedEnum {
 
 	PENDING("PEN"),
 	ACCEPTED("ACC"),
+	HOLD("HOL"),

```

Comment:
Is this supposed to be HOL or HLD?

Reply:
Although I agree that HLD sounds better, realized that we mostly use first three characters of the whole word as the short name.